drivers/project_scorecard/app/views/project_scorecard/warehouse_reports/scorecards/_ce_participation.haml

```diff
@@ -0,0 +1,52 @@
+%tr
+  %th{colspan: 12}
+    = _('Coordinated Entry System Particpation 20%')
+    %br
+    = simple_format(_('24 CFR 578.(a)(8) states that CoCs must establish and operate a Coordinated Entry System to allocate assistance as effectivley as possible, prioritizing services to those that need it the most.'))
+%tr
+  %th{colspan: 2}
+    Time to Lease Up
+  %th{colspan: 2}
+    = simple_format(_('TX-601 established Community-wide performance expectations in 2016 and identified that well-functioning Housing Crisis Response System moves households from homelessness to permanent housing in less than 30 days.'))
+    %br
+    Sage APR Q22c
+  %th{colspan: 3}
+    Average Time to Lease Up
+  %td{colspan: 1}
+    = f.input :days_to_lease_up, label: false, disabled: @report.locked?(:days_to_lease_up, current_user)
```

Comment:
Instead of disabling the input, let's put the value (or in the case of a select, the option text) as text.